src/core/components/courses/CourseHistory.vue

```diff
@@ -99,19 +99,28 @@ export default {
       return { pre: 'Nouveau', type: 'créneau', post: 'le', infos };
     },
     getSlotCreationDetails () {
-      return get(this.courseHistory, 'slot.address.fullAddress', 'Pas d\'adresse renseignée.');
+      if (get(this.courseHistory, 'slot.address.fullAddress')) {
+        return get(this.courseHistory, 'slot.address.fullAddress');
+      }
+      if (get(this.courseHistory, 'slot.meetingLink')) return get(this.courseHistory, 'slot.meetingLink');
+
+      return 'Pas d\'adresse renseignée.';
```

Comment:
Comme on a soit l'adresse, soit le lien, soit rien, que penses-tu de retourner directement comme suit : 
`return get(this.courseHistory, 'slot.address.fullAddress') || get(this.courseHistory, 'slot.meetingLink') || 'Pas d\'adresse renseignée.'` ?

Reply:
done